cmake/LibFuzzer.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+include(CTest)
```

Comment:
why is this stuff in the PR? Seems like something may be botched in your change set.

Reply:
This must have happened when I rebased. Investigating now ...

Reply:
Yeah I can't figure this one out. Going through each commit these files do not seem to show up. No clue why they appear when viewing "all commits".

Reply:
It’s because you based your branch on something other than master, most likely you branched from One of Colden’s branches

Reply:
Up at the top it does say "commits into master" so I'm not really sure how it could have been based off of one of Colden's branch. Colden investigated and couldn't see these files show up in the git diff itself. So it may be a github specific bug